docs/packaging-applications/build-servers/jenkins.md

```diff
@@ -168,3 +168,11 @@ This allows you to capture all related details and create clear traceability bet
 As an example here is what build information looks like when attached to a release:
 
 ![Build information on release page](images/build-information-release.png)
+
+## Contributing to the plugin
```

Comment:
It might be nice to include this section in a callout:
```suggestion
```success
**Contributing to the plugin**
```

Reply:
Also close the callout with the next suggestion below: